packages/public-api/src/middleware/common.ts

```diff
@@ -11,7 +11,7 @@ export const cors = createCorsMiddleware({
     if (originAllowList.length === 0 || (origin && originAllowList.includes(origin))) {
       callback(null, true);
     } else {
-      callback(new Error('Not allowed by CORS'));
+      callback(null, false);
```

Comment:
Are you sure it will work?

From docs:
<img width="753" alt="image" src="https://user-images.githubusercontent.com/5869473/198125538-d9b4559c-3506-46ae-b51c-3395ab1ae421.png">

Reply:
Yes, I'm fairly sure. Per said docs: 
```
Function - set origin to a function implementing some custom logic. The function takes the request origin as the first parameter and a callback (called as callback(err, origin), where origin is a non-function value of the origin option) as the second.
```